### ISSUE TITLE
Load More

### ISSUE THREAD

**Issue by kinju4374** [brookshi/LLMListView#24]
HI, I want to confirm whether Load More will help me or not. I have more than 4000 records in Database in UWP and I want to display about 10-15 records at a time and when user scrolls it should "Load more". Will your **LLMListView Load More** functionality help in this case?

**Comment by brookshi** [comment]
Yes, but you need add your 'Load More Data' code, please take a look at the load more sample.
